examples/projections/table/README.txt

```diff
@@ -19,64 +19,85 @@ The below table shows the projection codes for the 31 GMT projections.
    * - PyGMT Projection Argument
      - Projection Name
    * - **A**\ |lon0|/|lat0|\ [/\ *horizon*]/\ *width*
-     - Lambert azimuthal equal area
+     - :ref:`Lambert azimuthal equal area
+       <sphx_glr_projections_azim_azim_lambert.py>`
```

Comment:
I'm wondering is it better to use
```
:doc:`Lambert azimuthal equal area </projections/azim/azim_equidistant>`
```

Reply:
I made the change for this first reference to preview how it would look.

Reply:
I think the appearance is slightly nicer when referencing to the section rather than the document, but the doc reference structure would be more obvious for future maintenance of the table

Reply:
> I think the appearance is slightly nicer when referencing to the section rather than the document

I don't see any differences between these two references.

Reply:
This:
https://pygmt-g7gncwchs-gmt.vercel.app/projections/azim/azim_equidistant.html
versus this:
https://pygmt-g7gncwchs-gmt.vercel.app/projections/azim/azim_equidistant.html#sphx-glr-projections-azim-azim-equidistant-py

It is probably so minor that it does not matter (equal-area currently links to equidistant - I will change this once it's decided whether to use :doc: or :ref:)

Reply:
I'm OK with both but prefer `:doc:`

Reply:
OK, will update to ``:doc:``

Reply:
All are now `:doc:`